packages/core/src/browser/shell/application-shell.ts

```diff
@@ -1233,7 +1233,8 @@ export class ApplicationShell extends Widget {
         this.checkActivation(widget);
         Saveable.apply(
             widget,
-            () => this.widgets.filter((maybeSaveable): maybeSaveable is Widget & SaveableSource => !!Saveable.get(maybeSaveable)),
+            () => this.saveResourceService.autoSave !== 'off',
+            () => this.widgets.filter(maybeSaveable => !!Saveable.get(maybeSaveable)),
```

Comment:
We now have code in  `save-resource-service.ts` to track saveable widgets. Why do we still need this parameter?